common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -77,7 +80,15 @@ public MojoScorer(
     this.contributionResponseConverter = contributionResponseConverter;
     this.modelInfoConverter = modelInfoConverter;
     this.csvConverter = csvConverter;
-    pipelineShapley.setShapPredictContrib(true);
+
+    if (ENABLE_SHAPLEY_CONTRIBUTION) {
+      // note the mojo pipeline need to be reloaded here as we have a constrain from java mojo
+      // both SHAP values and predictions cannot be provided with the same pipeline
+      // Link: https://github.com/h2oai/mojo2/blob/7a1ab76b09f056334842a5b442ff89859aabf518/doc/shap.md
+
+      pipelineTransformedShapley = loadMojoPipelineFromFile();
+      pipelineTransformedShapley.setShapPredictContrib(true);
```

Comment:
Is this constructor called more than once during the life of a single scorer run?  Just checking to make sure that the MOJO isn't loaded more than once.

Reply:
Its called only once from the local rest scorer from the ScorerConfiguration.. But it is also called once from other scorers like gcp's vertex ai and sagemaker.